Running elevation on successful server produce unclear output

After running `elevate-cpanel --start` on an already successfully updated system the display output is unclear

We should say that the elevation process is over
and do not display the log as this is the case currently 

```
[root@elevation-1 ~]# /scripts/elevate-cpanel --start
Your elevation update is already in progress (stage 6).
You can monitor the current update by running:

    /scripts/elevate-cpanel

* 14-18:10:30 (272) [INFO] # Monitoring existing upgrade (stage=6) process via: tail -f /var/log/elevate-cpanel.log
Running transaction
  Preparing        :                                                        1/1
  Upgrading        : cloudlinux-linksafe-1-1.24.el8.noarch                  1/6
  Running scriptlet: cloudlinux-linksafe-1-1.24.el8.noarch                  1/6
  Upgrading        : alt-python38-setuptools-wheel-54.1.2-1.el8.noarch      2/6
  Upgrading        : alt-python38-pip-wheel-20.2.4-1.el8.noarch             3/6
  Running scriptlet: cloudlinux-linksafe-1-1.23.el7.noarch                  4/6
  Cleanup          : cloudlinux-linksafe-1-1.23.el7.noarch                  4/6
  Running scriptlet: cloudlinux-linksafe-1-1.23.el7.noarch                  4/6
  Cleanup          : alt-python38-setuptools-wheel-54.1.2-1.el7.noarch      5/6
  Cleanup          : alt-python38-pip-wheel-20.2.4-1.el7.noarch             6/6
  Running scriptlet: cloudlinux-linksafe-1-1.24.el8.noarch                  6/6
  Verifying        : alt-python38-pip-wheel-20.2.4-1.el8.noarch             1/6
  Verifying        : alt-python38-pip-wheel-20.2.4-1.el7.noarch             2/6
  Verifying        : alt-python38-setuptools-wheel-54.1.2-1.el8.noarch      3/6
  Verifying        : alt-python38-setuptools-wheel-54.1.2-1.el7.noarch      4/6
  Verifying        : cloudlinux-linksafe-1-1.24.el8.noarch                  5/6
  Verifying        : cloudlinux-linksafe-1-1.23.el7.noarch                  6/6

Upgraded:
  alt-python38-pip-wheel-20.2.4-1.el8.noarch
  alt-python38-setuptools-wheel-54.1.2-1.el8.noarch
  cloudlinux-linksafe-1-1.24.el8.noarch

Complete!

* 14-16:55:50 (1076) [INFO] ******************************************************************************************
* 14-16:55:50 (1077) [INFO] *
* 14-16:55:50 (1078) [INFO] * Great SUCCESS! Your upgrade to AlmaLinux 8 is complete.
* 14-16:55:50 (1079) [INFO] *
* 14-16:55:50 (1080) [INFO] ******************************************************************************************
* 14-16:55:51 (472) [ERROR] Sending notification: Successfully update to AlmaLinux 8
* 14-16:55:51 (473) [ERROR] The cPanel & WHM server has completed the elevation process from CentOS 7 to AlmaLinux 8.
* 14-16:55:51 (1076) [INFO] ******************************************************************************************
* 14-16:55:51 (1077) [INFO] *
* 14-16:55:51 (1078) [INFO] * Doing final reboot
* 14-16:55:51 (1079) [INFO] *
* 14-16:55:51 (1080) [INFO] ******************************************************************************************
* 14-16:55:51 (2902) [INFO] Running: /usr/sbin/reboot now

```

--- a/elevate-cpanel
+++ b/elevate-cpanel
@@ -277,15 +277,22 @@ sub monitor_upgrade {
 sub start($self) {
     my $stage = get_stage();
     if ( $stage != 0 ) {
-        warn <<"EOS";
-Your elevation update is already in progress (stage $stage).
-You can monitor the current update by running:
+        my $header;
+        if ( $stage > VALID_STAGES ) {
+            $header = q[The 'elevate-cpanel' script has already been executed on this server.];
+        }
+        else {
+            $header = qq[An elevation process is currently in progress: running stage $stage];
+        }
 
-    /scripts/elevate-cpanel
+        die <<~"EOS";
+        $header
+        You can check the log by running:
+            /scripts/elevate-cpanel --log
+        or check the elevation status:
+            /scripts/elevate-cpanel --check
+        EOS
 
-EOS
-
-        return 0;
     }
 
     system touch => LOG_FILE;

